common/types/shared.go

```diff
@@ -63,6 +63,21 @@ func (v *ActivityTaskCancelRequestedEventAttributes) GetActivityID() (o string)
 	return
 }
 
+// Size returns the approximate memory used in bytes
+func (v *ActivityTaskCancelRequestedEventAttributes) Size() uint64 {
+	if v == nil {
```

Comment:
A question: Can we generate this code (with gowrap or something else) for all the structs instead of writing it manually?